packages/twenty-front/src/modules/auth/components/Logo.tsx

```diff
@@ -1,67 +1,85 @@
+import { useTheme } from '@emotion/react';
 import styled from '@emotion/styled';
 
 import { getImageAbsoluteURI } from '~/utils/image/getImageAbsoluteURI';
+import { isDefined } from '~/utils/isDefined';
 
 type LogoProps = {
   workspaceLogo?: string | null;
+  size?: string | null;
+  includeTwentyLogo?: boolean;
```

Comment:
Took me a minute to understand what this was. Maybe name `overlayTwentyLogoInCorner` - very verbose but at least we'd understand 😅

Reply:
It changes in the last commit